Pipfile

```diff
@@ -0,0 +1,17 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+jinja2 = "*"
+gunicorn = "*"
+flask = "*"
+pytest = "*"
```

Comment:
pytest and pytest-cov should be in the dev packages

Reply:
Updated pipfile